app/services/puppet_fact_parser.rb

```diff
@@ -140,6 +140,21 @@ def cores
     facts.dig('processors', 'count') || facts['processorcount']
   end
 
+  def disks_total
+    disks      = facts['disks']
+    disks_size = 0
+
+    return nil unless disks
+
+    if disks.any?
+      disks.values.each do |disk_attrs|
+        disks_size += disk_attrs&.fetch('size_bytes', 0).to_i
+      end
```

Comment:
I think this should also work:
```suggestion
    disk_size = disks.sum { |disk| disk&.fetch('size_bytes', 0).to_i }
```